binding/vue/README.md

```diff
@@ -52,154 +47,198 @@ Signup or Login to [Picovoice Console](https://console.picovoice.ai/) to get you
 
 ## Installation
 
-Install the package using `npm` or `yarn`. You will also need to add `@picovoice/web-voice-processor` and one of the `@picovoice/rhino-web-**-worker` series of packages for the specific language model:
+Using `yarn`:
 
-E.g. English:
+```console
+yarn add @picovoice/rhino-vue @picovoice/web-voice-processor
+```
+
+or using `npm`:
 
 ```console
-yarn add @picovoice/rhino-web-vue @picovoice/rhino-web-en-worker @picovoice/web-voice-processor
+npm install --save @picovoice/rhino-vue @picovoice/web-voice-processor
 ```
+### AccessKey
```

Comment:
Duplicate AccessKey section (you may want to double check your other bindings for this)